raiden/tests/fixtures/variables.py

```diff
@@ -367,3 +367,8 @@ def skip_if_not_matrix(request):
     if request.config.option.transport in ('matrix', 'all'):
         return
     pytest.skip('This test works only with Matrix transport')
+
+
+@pytest.fixture
+def is_private_to_join_rule(room_privacy_settings):
```

Comment:
What is the purpose of this fixture?

Reply:
I use it to parametrize the different `is_private/join_rule` permutations that we want to test. I could squeeze that into the `private_rooms` fixture, but then the old tests have to be changed as well.

Reply:
But you can do that directly on the test, you don't need this 'dummy' fixture for that. e.g. you can 'invent' parameters on any test via `parametrize`